lib/core/prometheusMetrics.hpp

```diff
@@ -16,6 +16,116 @@ using std::string;
 
 namespace kotekan {
```

Comment:
I wonder if this should have it's own namespace.  Just because `kotekan::Family` and `kotekan::Metric` seems overly generic.   I would either subname space this, or add some indication this is for Prometheus to all the names.

Reply:
I'm inclined to go with the subname: either `kotekan::prometheus::` or `kotekan::metrics`. It would probably make sense to move the `prometheusMetrics` class into it as well, no?

Reply:
I would go with `kotekan::prometheus::`, the word "metrics" might well apply to other systems of tracking stats.

Reply:
Sounds good. And now `kotekan::prometheusMetrics` becomes `kotekan::prometheus::Metrics`.